sql/handler.h

```diff
@@ -1061,6 +1061,18 @@ struct handlerton
   */
   bool (*rotate_encryption_master_key)(void);
 
+ /**
+    @brief
+    Initiate key rotation of specific tablespaces of an engine
```

Comment:
IMHO not perfectly clear - what specific tablespaces? I'd try calling the handlerton fixup_tablespace_keys or similar - to show that the rotation happens from "incorrect" to "correct"

Reply:
changed API name to fix_tablespaces_empty_uuid()